Spigot-Server-Patches/0384-Anti-Xray.patch

```diff
@@ -38,6 +39,7 @@ index 48676152152faf7a7b9524ac37d8b4a8c32c4e2c..ca2ac17747d4f2acf1df056759c5c182
 +        maxChunkSectionIndex = getInt("anti-xray.max-chunk-section-index", 3);
 +        maxChunkSectionIndex = maxChunkSectionIndex > 15 ? 15 : maxChunkSectionIndex;
 +        updateRadius = getInt("anti-xray.update-radius", 2);
++        lavaObscures = getBoolean("anti-xray.lava-obscures", true);
```

Comment:
Is changing default behaviour after so many versions a good idea?

Reply:
Aikar on Discord: lava-obscures: true, default on is reasonable here

Reply:
Yes, defaults represent what majority will want. majority will clearly want this.